.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,29 @@
+name: Build for Docker
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
```

Comment:
We've been using explicit runner versions (`ubuntu-22.04`) for all of our workflows to control when and how they update (they've broken on us during alias updates in the past)

Reply:
Will remove for now